src/Microsoft.Azure.WebJobs.Extensions.Dapr/DaprExtensionConfigProvider.cs

```diff
@@ -200,6 +200,26 @@ static DaprBindingMessage CreateBindingMessage(JToken arg)
         }
 
         static DaprBindingMessage CreateBindingMessage(JsonElement jsonElement)
+        {
+            if (jsonElement.ValueKind == JsonValueKind.String)
+            {
+                byte[] jsonBytes = Encoding.UTF8.GetBytes(jsonElement.GetString());
+                Utf8JsonReader reader = new Utf8JsonReader(jsonBytes);
+
+                if (JsonElement.TryParseValue(ref reader, out JsonElement? jsonElementObj) && jsonElementObj != null)
+                {
+                    return GetDaprBindingMessageObj(jsonElementObj.Value);
+                }
+                else
+                {
+                    throw new ArgumentException("Could not read jsonElement parameter.", nameof(jsonElement));
```

Comment:
Can we also test to cover this for completeness?

Reply:
Done!